### DISCUSSION TITLE
Fulfillment Inbound API v2024-03-20   getInboundPlan ERROR

### DISCUSSION THREAD

**Discussion by gavinjzw** [amzn/selling-partner-api-models#4085] [Q&A]
{
  "errors": [
    {
      "code": "BadRequest",
      "message": "ERROR: GetInboundPlan is not supported for the cross border Amazon Global Logistics program.",
      "details": ""
    }
  ]
}

AGL InboundPlan request API error, may I ask which API should be modified to access for AGL??

**Comment by kmander** [comment]
Did you find the answer @gavinjzw ?

**Comment by puppsupr** [comment]
Hi,
Did you create this inbound plan using the new v2024 APIs ?
